service/src/test/java/uk/gov/hmcts/reform/fpl/service/PlacementServiceTest.java

```diff
@@ -446,6 +449,58 @@ void shouldFetchPlacementFeeWhenNoPreviousPlacementPayment() {
             verify(feeService).getFeesDataForPlacement();
             verifyNoInteractions(time);
         }
+
+        @Test
+        void shouldReturnAllChildPlacementApplications() {
+            UUID placementId = randomUUID();
+            Element<Child> childForPlacement = testChild();
+            List<Element<String>> childPlacementOrders = underTest.getPlacements(CaseData.builder()
```

Comment:
it is a little bit of nightmare to read this ;) Maybe CaseData can have dedicated field?